kusama-guide/pages/en/index.js

```diff
@@ -1,19 +1,110 @@
 const React = require('react');
 
-const Index = () => {
+const translate = require('../../server/translate.js').translate;
 
-  return (
-    <div>
-      <script
-        dangerouslySetInnerHTML={{
-            __html: `
-          window.location.href = 'docs/en/kusama-index';
-        `,
-        }}
-      />
-      If you are not redirected automatically, follow this <a href="docs/en/kusama-index/">link</a>.
-    </div>
-  );
+class HomeSplash extends React.Component {
+  render() {
+    // const {siteConfig, language = ''} = this.props;
```

Comment:
```suggestion
    // const {siteConfig, language = ''} = this.props;
```

Remove if not needed